platforms/native/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftIncrementalBuildIntegrationTest.groovy

```diff
@@ -364,23 +361,10 @@ class SwiftIncrementalBuildIntegrationTest extends AbstractInstalledToolChainInt
         for (SourceFile sourceFile : sourceElement.getFiles()) {
             def swiftFile = file("src", sourceSetName, sourceFile.path, sourceFile.name)
             result.add(objectFileFor(swiftFile, intermediateFilesDirPath).relativizeFrom(intermediateFilesDir).path)
-            result.add(swiftmoduleFileFor(swiftFile).relativizeFrom(intermediateFilesDir).path)
```

Comment:
Don't try to calculate the exact set of `swiftc` version specific output files, and just make sure there is an object file and the dependency files.